flowers.json

```diff
@@ -226,7 +226,7 @@
       ],
       "blooming_period": "--07-20/--10-31",
       "description": {
-        "summary": "Solidago rigida, or stiff goldenrod, is an upright perennial with yellow or orange flowers. It is often used in sunny gardens, prairie plantings or restorations. Besides bees, the flowers attract a wide variety of flower-visiting insects. The Ojibwe traditionally used the root as an enema and took an infusion of the root to treat 'stoppage of urine.' The Meskwaki traditionally made the flowers into a lotion and used them on bee stings and for swollen faces.",
+        "summary": "Stiff goldenrod is an upright perennial with yellow or orange flowers. It is often used in sunny gardens, prairie plantings or restorations. Besides bees, the flowers attract a wide variety of flower-visiting insects. The Ojibwe traditionally used the root as an enema and took an infusion of the root to treat 'stoppage of urine.' The Meskwaki traditionally made the flowers into a lotion and used them on bee stings and for swollen faces.",
```

Comment:
We should use double quotes rather than single quotes (and we should probably use the proper typographic opening and closing quotes.